src/components/breadcrumb/breadcrumb-step/breadcrumb-step.js

```diff
@@ -2,10 +2,15 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import SvgIcon from '@material-ui/core/SvgIcon';
 import Grid from '@material-ui/core/Grid';
 import Fab from '@material-ui/core/Fab';
 import styles from '../breadcrumb.module.css';
+import QuestionMarkIcon from '../../icons/question-mark';
+import CheckmarkIcon from '../../icons/checkmark';
+import DollarSignIcon from '../../icons/dollar-sign';
+import NotesIcon from '../../icons/notes';
+import Notes2Icon from '../../icons/notes-2';
+import LocationIcon from '../../icons/location';
```

Comment:
We should probably have an index in the `icons` directory that exports all of the icons. That way we can import them all at one, i.e.:

``` javascript
import {
    QuestionMarkIcon,
    CheckmarkIcon,
    DollarSignIcon,
    NotesIcon,
    Notes2Icon,
    LocationIcon
} from '../../icons;
```

Reply:
Good idea I'll do that